quiz-app/components/Question/index.js

```diff
@@ -1,7 +1,33 @@
-export default function Question({text}) {
+export default function Question({ question, onChange }) {
+  const buttonTypes = ["agree", "slightlyAgree", "slightlyDisagree", "disagre"];
```

Comment:
I think we need to add the neutral option here too. 

Flagging a typo: disagree is missing an 'e'. 

Suggestion: Can we refactor each name in the buttonTypes array to remove the camel-case formatting so "Slightly Agree" renders instead of "slightlyAgree"?